src/traces/isosurface/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Isosurface = {};
+
+Isosurface.attributes = require('./attributes');
+Isosurface.supplyDefaults = require('./defaults');
+Isosurface.calc = require('./calc');
+Isosurface.colorbar = {
+    min: 'cmin',
+    max: 'cmax'
+};
+Isosurface.plot = require('./convert');
+
+Isosurface.moduleType = 'trace';
+Isosurface.name = 'isosurface',
+Isosurface.basePlotModule = require('../../plots/gl3d');
+Isosurface.categories = ['gl3d'];
+Isosurface.meta = {
+    description: [
```

Comment:
This thing looks outdated. I don't see attributes `width`, `height` and `depth` anywhere.